packages/twenty-server/src/modules/messaging/message-import-manager/drivers/exceptions/message-import-driver.exception.ts

```diff
@@ -12,6 +12,8 @@ export enum MessageImportDriverExceptionCode {
   TEMPORARY_ERROR = 'TEMPORARY_ERROR',
   INSUFFICIENT_PERMISSIONS = 'INSUFFICIENT_PERMISSIONS',
   UNKNOWN = 'UNKNOWN',
+  UNAUTHORIZED = 'UNAUTHORIZED',
+  FORBIDDEN = 'FORBIDDEN',
```

Comment:
forbidden and unauthorized fall under INSUFFICIENT_PERMISSIONS in our current implementation